.NET/Microsoft.Recognizers.Text.Number/Hindi/Parsers/HindiNumberParserConfiguration.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Immutable;
+using System.Globalization;
+using System.Text.RegularExpressions;
+using Microsoft.Recognizers.Definitions.Hindi;
+
+namespace Microsoft.Recognizers.Text.Number.Hindi
+{
+    public class HindiNumberParserConfiguration : BaseNumberParserConfiguration
+    {
+        private const RegexOptions RegexFlags = RegexOptions.Singleline | RegexOptions.ExplicitCapture;
+
+        public HindiNumberParserConfiguration(INumberOptionsConfiguration config)
+        {
+
+            this.Config = config;
+            this.LangMarker = NumbersDefinitions.LangMarker;
+
+            // @TODO Temporary workaround
```

Comment:
Can you describe what this is working around? This way others can suggest possible approaches.

Reply:
remnant from English original file, removed